plugins/websocket_server/WebsocketServer.cc

```diff
@@ -748,6 +748,37 @@ void WebsocketServer::OnMessage(int _socketId, const std::string &_msg)
     this->QueueMessage(this->connections[_socketId].get(),
         data.c_str(), data.length());
   }
+  /// \todo(nkoeng) Deprecate this in Ignition Fortress, and instruct users
+  /// to rely on the "scene" message.
+  else if (frameParts[0] == "particle_emitters")
+  {
+    igndbg << "Particle emitter request recieved for world["
```

Comment:
```suggestion
    igndbg << "Particle emitter request received for world["
```

Reply:
Fixed in c404a076c9d40561bafa4371e4b2f2e7225ed4cf